src/wasm-lib/src/wasm.rs

```diff
@@ -576,3 +577,26 @@ pub fn base64_decode(input: &str) -> Result<Vec<u8>, JsValue> {
 
     Err(JsValue::from_str("Invalid base64 encoding"))
 }
+
+#[wasm_bindgen]
+pub struct WasmCircleParams {
+    pub center_x: f64,
+    pub center_y: f64,
+    pub radius: f64,
+}
+
+/// Calculate a distance.
```

Comment:
Ugh, I'll fix this tomorrow

Reply:
I'm confused what needed fixing?

Reply:
The comment is wrong; it should be talking about calculating the center and radius from the 3 points :p